src/components/UI/ChainSelect/ChainSelect.js

```diff
@@ -1,12 +1,13 @@
 import {Select, MenuItem, FormControl} from '@mui/material';
 import {ChainInfo, ChainType} from '@starkware-industries/commons-js-enums';
+import {openInNewTab} from '@starkware-industries/commons-js-utils';
```

Comment:
Why 2 lines of imports from the same library?

Reply:
Bad merging, fixed.